src/scene-entry-manager.js

```diff
@@ -218,7 +218,8 @@ export default class SceneEntryManager {
     });
 
     document.addEventListener("paste", e => {
-      if (e.target.nodeName === "INPUT" && document.activeElement === e.target) return;
+      if ((e.target.nodeName === "INPUT" || e.target.nodeName === "TEXTAREA") && document.activeElement === e.target)
```

Comment:
`e.matches("input, textarea")` would be a bit cleaner for the first part of this. (might technically be slower but this isn't hot code)